modules/ROOT/pages/exchange-api.adoc

```diff
@@ -149,6 +153,37 @@ If the asset is published asynchronously, then a response like this is returned:
 
 The endpoint `publicationStatusLink` provides the asset publication status.
 
+[[set-mutable-data]]
+=== Set Mutable Data
+
+This example shows how to create your asset and set the mutable data describing it in the same request. The mutable data of an asset includes categories, custom fields, tags, and documentation pages.
+
+This example publishes the category and custom field, which have the keys `categoryKey` and `fieldKey`. They must be configured for your organization. Publication fails if they are not configured for your organization or if they are not deleted from the pom.
+
+It is not possible to submit new mutable data to an existing patch version such as 1.2.3. Instead, create a new patch version such as 1.2.4 with updated mutable data such as categories.
```

Comment:
Could we be more specific? like:
```suggestion
This is an endpoint for asset creation, therefore if the asset already exists the request will fail with a 409 (Conflict), if you only want to modify the mutable data you will have to change the version of the asset, to publish a new one.
```

Maybe the last part is not necessary since we are going to have another endpoint to do that in the short future.

Reply:
Yes, that's better! We'll use this for now and update it when we have the other endpoint.